crawl-ref/source/mon-spell.h

```diff
@@ -1695,6 +1704,16 @@ static const mon_spellbook mspell_list[] =
       }
     },
 
+    {  MST_CIRI,
+      {
+       { SPELL_WARNING_CRY, 40, MON_SPELL_VOCAL | MON_SPELL_BREATH },
+       { SPELL_MALIGN_GATEWAY, 30, MON_SPELL_NATURAL },
+       { SPELL_FORCEFUL_INVITATION, 30, MON_SPELL_NATURAL },
+       { SPELL_PLANEREND, 20, MON_SPELL_NATURAL },
```

Comment:
I like the idea of having a synergistic trio of uniques who work together to fight the player, but I worry that having a strong focus on summons dilutes that concept. You're not fighting Ciri and Geralt and Yennefer, you're fighting a whole pile of guys, with Geralt and Yennefer floating somewhere in a sea of summons.

I get the thematic connection, but from a play perspective, I'd want to go in a different direction for Ciri's ability set. Ranged damage, hexes..?